pkg/types/helm/provenance_test.go

```diff
@@ -37,12 +38,16 @@ func TestProvenance(t *testing.T) {
 		t.Fatalf("unmarshal error: %v", err)
 	}
 
-	checksum, err := provenance.GetChartHash()
+	algorithm, checksum, err := provenance.GetChartAlgorithmHash()
 
 	if err != nil {
 		t.Fatalf("Error retrieving chart hash: %v", err)
 	}
 
+	if models.AlpineV001SchemaPackageHashAlgorithmSha256 != algorithm {
```

Comment:
You should be able to find this in helm somewhere instead of alpine...

Reply:
@dlorenc whoops. good catch. updated